src/Field/Field.tsx

```diff
@@ -32,16 +33,25 @@ export const Field = ({
   required,
   showCaret,
   fullHeight = false,
+  assistiveText,
   ...marginProps
 }: FieldProps) => {
   return (
     <Container className={className} {...marginProps}>
       {label && (
         <Box mb={{ custom: outlined ? 4 : 0 }}>
```

Comment:
Should we move this to the version with `renderAsTitle = false`? We already have some margin bottom when displayed as title

Reply:
resolved in [82e535](https://github.com/marshmallow-insurance/smores-react/pull/2118/commits/82e535d107f5efa170c85b575cc570d45c8c77aa)